python/pyspark/sql/catalog.py

```diff
@@ -258,6 +258,11 @@ def refreshTable(self, tableName):
         """Invalidate and refresh all the cached metadata of the given table."""
         self._jcatalog.refreshTable(tableName)
 
+    @since(2.1.1)
```

Comment:
Will change it to `2.1`. `2.1.1` causes the doc build break. If needed, I can do more investigation to see how to do it for `2.1.1`